.travis.yml

```diff
@@ -25,9 +28,6 @@ script:
   `make`. Please run it and commit the changes"; git status --porcelain; false; fi
 - make build-verify # verify that we set version on the packages built by bazel
 # The make bazel-test might take longer then the current timeout for a command in Travis-CI of 10 min, so adding a keep alive loop while it runs
```

Comment:
Maybe you should move the comment and change it to something like `several commands might take longer than ten minutes...`?